<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>845</width>
    <height>541</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>845</width>
    <height>541</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>單步執行器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/Image/singleopIcon24.png</normaloff>:/Image/singleopIcon24.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 235, 235);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 249, 249);</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>251</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="RegsLayout" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0,1,3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="RegsPCLable">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 214, 215);
border-left: 1px solid #989898;
border-top: 1px solid #989898;</string>
       </property>
       <property name="text">
        <string>PC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="RegsLLable">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 214, 215);
border-left: 1px solid #989898;
border-top: 1px solid #989898;</string>
       </property>
       <property name="text">
        <string>L</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="RegsTLable">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 214, 215);
border-left: 1px solid #989898;
border-top: 1px solid #989898;</string>
       </property>
       <property name="text">
        <string>T</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="RegsSLable">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 214, 215);
border-left: 1px solid #989898;
border-top: 1px solid #989898;</string>
       </property>
       <property name="text">
        <string>S</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="RegsALable">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 214, 215);
border-left: 1px solid #989898;
border-top: 1px solid #989898;</string>
       </property>
       <property name="text">
        <string>A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="RegsBLable">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 214, 215);
border-left: 1px solid #989898;
border-top: 1px solid #989898;</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="RegsXLable">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 214, 215);
border-left: 1px solid #989898;
border-top: 1px solid #989898;</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="RegsSWLable">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(210, 214, 215);
border-left: 1px solid #989898;
border-top: 1px solid #989898;
border-bottom: 1px solid #989898;
</string>
       </property>
       <property name="text">
        <string>SW</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="RegsXContentLable">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #fff;
border-right: 1px solid #989898;
border-top: 1px solid #989898;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="RegsAContentLable">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #fff;
border-right: 1px solid #989898;
border-top: 1px solid #989898;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="RegsPCContentLable">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #fff;
border-right: 1px solid #989898;
border-top: 1px solid #989898;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="RegsLContentLable">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #fff;
border-right: 1px solid #989898;
border-top: 1px solid #989898;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="RegsBContentLable">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #fff;
border-right: 1px solid #989898;
border-top: 1px solid #989898;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="RegsSContentLable">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #fff;
border-right: 1px solid #989898;
border-top: 1px solid #989898;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="RegsTContentLable">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #fff;
border-right: 1px solid #989898;
border-top: 1px solid #989898;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="RegsSWContentLable">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #fff;
border-right: 1px solid #989898;
border-top: 1px solid #989898;
border-bottom: 1px solid #989898;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="OpenFileLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>檔案位置</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="OpenFileTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 3px;
border-bottom: 1px solid #989898;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PMingLiU'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'微軟正黑體'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectFileBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#SelectFileBtn {
background-color: rgb(139, 139, 139);
color: #fff;
border: none;
border-radius: 5px;
}
QPushButton#SelectFileBtn::hover {
background-color:rgb(206, 206, 206);
color: rgb(45, 39, 39);
}
QPushButton#SelectFileBtn::pressed {
background-color: rgb(165, 165, 165);
border-bottom: 2px solid rgb(65, 65, 65);
}</string>
    </property>
    <property name="text">
     <string>選擇檔案</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="CurInstructLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>現在執行指令</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurInstructContentLable">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
border-bottom: 1px solid #989898;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="OpenFileStatusLable">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Image/FileNotSelected.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="OneStepBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#OneStepBtn {
background-color: rgb(139, 139, 139);
color: #fff;
border: none;
border-radius: 5px;
}
QPushButton#OneStepBtn::hover {
background-color:rgb(206, 206, 206);
color: rgb(45, 39, 39);
}
QPushButton#OneStepBtn::pressed {
background-color: rgb(165, 165, 165);
border-bottom: 2px solid rgb(65, 65, 65);
}</string>
    </property>
    <property name="text">
     <string>下一指令</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="MemeryDisplayPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>561</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>細明體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
</string>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RegsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>暫存器</string>
    </property>
   </widget>
   <widget class="QLabel" name="MemoryLable">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>記憶體</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveObpBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#RemoveObpBtn {
background-color: rgb(139, 139, 139);
color: #fff;
border: none;
border-radius: 5px;
}
QPushButton#RemoveObpBtn::hover {
background-color:rgb(206, 206, 206);
color: rgb(45, 39, 39);
}
QPushButton#RemoveObpBtn::pressed {
background-color: rgb(165, 165, 165);
border-bottom: 2px solid rgb(65, 65, 65);
}</string>
    </property>
    <property name="text">
     <string>移除Object Program</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AuthorSign">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Vivaldi</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(145, 145, 145);
border: none</string>
    </property>
    <property name="plainText">
     <string>Power by Huang Shih Hao..2015/6/11     https://github.com/dt9450108/SICXE_simulator_parted</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelIcon">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Image/rightUpIcon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="FullExecBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#FullExecBtn {
background-color: rgb(139, 139, 139);
color: #fff;
border: none;
border-radius: 5px;
}
QPushButton#FullExecBtn::hover {
background-color:rgb(206, 206, 206);
color: rgb(45, 39, 39);
}
QPushButton#FullExecBtn::pressed {
background-color: rgb(165, 165, 165);
border-bottom: 2px solid rgb(65, 65, 65);
}</string>
    </property>
    <property name="text">
     <string>輸出完整執行結果</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoadAddrLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>載入記憶體位置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoadAddrLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 3px;
border-bottom: 1px solid #989898;</string>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>開啟檔案</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>結束程式</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
